students/dodam/westagram/users/views.py

```diff
@@ -10,24 +10,34 @@
 class UserView(View):
 	def post(self,request):
 		try:	
-			data = json.loads (request.body) # 프론트가 requset한 정보를 바디에 담아 json 을 활용해 읽어내겠다는 의미 
-			# 중복에러 설정하기
+			data = json.loads(request.body)  
 			if User.objects.filter(name=data['name']).exists() or User.objects.filter(email=data['email']).exists(): 			
 				return JsonResponse({'MESSAGE': 'DATA_OVERLAP'}, status = 400)		
 
-			# 이메일 필수조건 설정 및 패스워드 설정 관련 정규식 : 8자이상, 문자, 숫자, 특수문자 조건 충족
 			if ('@' not in data['email']) or ('.'not in data['email']) or (not password_regular.match(data['password'])):
 				return JsonResponse ({'MESSAGE': 'INVALID_FORMAT'}, status = 400)
 
-
 			User.objects.create(
 			name 		 = data['name'],
 			email	 	 = data['email'],
 			password	 = data['password'],
 			phone_number = data['phone_number'], 
 			birthday 	 = data['birthday'],
 			)
-			return JsonResponse ({'MESSAGE' : 'SUCCESS'}, status = 201)
-
-		except KeyError: # 키 값이 빠진다면 나오는 오류
-			return JsonResponse ({'MESSAGE': 'KEY_ERROR'}, status = 400)
\ No newline at end of file
+			return JsonResponse ({'MESSAGE':'SUCCESS'}, status = 201)
+
+		except KeyError: 
+			return JsonResponse ({'MESSAGE':'KEY_ERROR'}, status = 400)
+
+class LoginView(View):
+	def post(self,requset):
+		try:
+			data = json.loads(requset.body)
+			if (data['email']) or (data['password']) == "" :
+				return JsonResponse ({'MESSAGE': 'KEY_ERROR'},status = 400)	
+			if not (User.objects.filter(data['email']).exists()) or data['password'] == (User.objects.filter(data['password'])):
+				return JsonResponse ({'MESSAGE':'INVALID_USER'}, status =401)
+			return JsonResponse ({'MESSAGE':'SUCCESS'}, status = 200)
```

Comment:
```suggestion
			data = json.loads(requset.body)
			
			if (data['email']) or (data['password']) == "" :
				return JsonResponse ({'MESSAGE': 'KEY_ERROR'},status = 400)
					
			if not (User.objects.filter(data['email']).exists()) or data['password'] == (User.objects.filter(data['password'])):
				return JsonResponse ({'MESSAGE':'INVALID_USER'}, status =401)
			return JsonResponse ({'MESSAGE':'SUCCESS'}, status = 200)
```
1. 가독성을 위해 로직이 달라지면 줄을 띄워주세요.
2. email이나 password 자체가 들어오지 않는 경우가 key error이고 해당 값이 빈 값으로 들어온 경우는 잘못된 값이 들어왔다는 에러 메세지가 적절하겠죠?
3. 유저가 존재하는지 먼저 판단 후 해당 유저의 비밀번호가 들어온 비밀번호와 일치하는지 확인해야합니다. 두개의 로직으로 분리해주세요.
4. User.objects.filter(data['email'])이 제대로 동작 하는지, 동작한다면 어떤 값을 반환하는지 확인해주세요